### PR TITLE
Import initial code

### PR DESCRIPTION
**Pull request by bessbd**
This change does the initial code import.

### CODE REVIEW

File: cloudantv1/cloudant_v1_integration_test.go [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+import (
+	"fmt"
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/IBM/go-sdk-core/v4/core"
+	"github.ibm.com/cloudant/cloudant-go-sdk/cloudantv1"
```
Comment (emlaver) [inline cloudantv1/cloudant_v1_integration_test.go:25]:
```suggestion
	"github.com/IBM/cloudant-go-sdk/cloudantv1"
```

File: examples/src/create_db_and_doc/create_db_and_doc.go [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.ibm.com/cloudant/cloudant-go-sdk/cloudantv1"
```
Comment (emlaver) [inline examples/src/create_db_and_doc/create_db_and_doc.go:21]:
```suggestion
	"github.com/IBM/cloudant-go-sdk/cloudantv1"
```

File: go.mod [hunk 3]
```diff
@@ -0,0 +1,11 @@
+module github.ibm.com/cloudant/cloudant-go-sdk
```
Comment (emlaver) [inline go.mod:1]:
```suggestion
module github.com/IBM/cloudant-go-sdk
```

File: examples/src/delete_doc/delete_doc.go [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+
+	"github.ibm.com/cloudant/cloudant-go-sdk/cloudantv1"
```
Comment (emlaver) [inline examples/src/delete_doc/delete_doc.go:20]:
```suggestion
	"github.com/IBM/cloudant-go-sdk/cloudantv1"
```

File: examples/src/get_info_from_existing_database/get_info_from_existing_database.go [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/IBM/go-sdk-core/v4/core"
+	"github.ibm.com/cloudant/cloudant-go-sdk/cloudantv1"
```
Comment (emlaver) [inline examples/src/get_info_from_existing_database/get_info_from_existing_database.go:22]:
```suggestion
	"github.com/IBM/cloudant-go-sdk/cloudantv1"
```

File: examples/src/update_doc/update_doc.go [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.ibm.com/cloudant/cloudant-go-sdk/cloudantv1"
```
Comment (emlaver) [inline examples/src/update_doc/update_doc.go:21]:
```suggestion
	"github.com/IBM/cloudant-go-sdk/cloudantv1"
```

File: cloudantv1/cloudant_v1_integration_test.go (hunk 1 above)
Comment (bessbd) [reply cloudantv1/cloudant_v1_integration_test.go]: Thank you for the suggestion. Committed.

**Review by ricellis — APPROVED** [review]

**Comment by bessbd** [comment]
Thank you for the review, @emlaver and @ricellis . I'm about to merge this.

**Pull request merged**